R/flea_starters.R

```diff
@@ -27,44 +27,56 @@ ff_starters.flea_conn <- function(conn, week = 1:17, ...) {
 }
 
 .flea_starters <- function(week, game_id, conn) {
+  cols <- c(injurytypeAbbreviaition = NA, injurytypeFull = NA, injurydescription = NA, injuryseverity = NA)
```

Comment:
I'd rather see this as an empty data.frame that gets `bind_rows()` on - and also I think it should use snake_case as per styling standards. 

```suggestion
  injury_cols <- data.frame(
    injury_type_abbr = character(),
    injury_type = character(),
    injury_desc = character(),
    injury_severity = character()
   )
```